comment_generation/app.py

```
@@ -36,15 +38,59 @@ def get_openai_api_key():
+        input=prompt,
+        instructions=instruction
+    )
+    return getattr(response, 'output_text', None)
+
+def call_xai_api(prompt: str, instruction: str):
+    chat = xai_client.chat.create(model="grok-4-1-fast-non-reasoning")
+    chat.append(system(instruction))
+    chat.append(user(prompt))
+    return chat.sample().content
+
+def generate_comment(prompt: str, instruction: str, api: str, headers: str):
```

Ruff (0.14.13)

Unused function argument: `headers`

(ARG001)